### PR TITLE
Add days_active_bits to clients_last_seen_v2

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
This PR adds the bit pattern required to implement DAU at client_level for Desktop. See [DENG-2975](https://mozilla-hub.atlassian.net/browse/DENG-2975).

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

[DENG-2975]: https://mozilla-hub.atlassian.net/browse/DENG-2975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3122)

### CODE REVIEW

**Review by kwindau — APPROVED** [review]

**Comment by scholtzan** [comment]
`https://github.com/mozilla/bigquery-etl/blob/616c495ee7ebd33e4951072429140c26c761bdd5/sql/moz-fx-data-shared-prod/telemetry/clients_last_seen/view.sql#L7` seems to still point to `clients_last_seen_v1`. Is that something that should be changed so that we can take advantage of the `active` bits in Looker and elsewhere?

**Review by scholtzan — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/query.sql [hunk 1]
```diff
@@ -6,6 +6,12 @@ WITH _current AS (
     -- 28 days for each usage criterion as a single 64-bit integer. The
     -- rightmost bit represents whether the user was active in the current day.
     CAST(TRUE AS INT64) AS days_seen_bits,
+    CAST(active_hours_sum > 0 AS INT64) & CAST(
+      COALESCE(
+        scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
+        scalar_parent_browser_engagement_total_uri_count_sum
+      ) AS INT64
```
Comment (ANich) [inline sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/query.sql:13]: Should this be `>=` some number?
```suggestion
      COALESCE(
        scalar_parent_browser_engagement_total_uri_count_normal_and_private_mode_sum,
        scalar_parent_browser_engagement_total_uri_count_sum
      ) >= 1 AS INT64
```

**Review by ANich — APPROVED** [review]
Left one question

**Comment by ANich** [comment]
> https://github.com/mozilla/bigquery-etl/blob/616c495ee7ebd33e4951072429140c26c761bdd5/sql/moz-fx-data-shared-prod/telemetry/clients_last_seen/view.sql#L7
> seems to still point to `clients_last_seen_v1`. Is that something that should be changed so that we can take advantage of the `active` bits in Looker and elsewhere?

I've rebased https://github.com/mozilla/bigquery-etl/pull/4238. There's a couple downstream tables that could be switched directly to `telemetry.clients_last_seen_v1` until that view (and table) are deprecated.

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/schema.yaml [hunk 2]
```diff
@@ -2346,3 +2346,6 @@ fields:
   - name: value
     type: INTEGER
     mode: NULLABLE
+- name: days_active_bits
```
Comment (kik-kik) [inline sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/schema.yaml:2349]: Should we add a description to this field to indicate what is meant by `active`?

Comment (lucia-vargas-a) [reply sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/schema.yaml]: I added a description for both days_seen_bits  and days_active_bits.

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/query.sql (hunk 1 above)
Comment (lucia-vargas-a) [reply sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_v2/query.sql]: > 0 good catch, TY!

**Pull request merged**